pkgs/tools/misc/tdfgo/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, buildGo118Module, fetchFromGitHub }:
+
+buildGo118Module rec {
+  pname = "tdfgo";
+  version = "v0.0.0-9f0b3315eed32409639a05aca55d7a0252681193";
+  src = fetchFromGitHub {
+    owner = "digitallyserviced";
+    repo = "tdfgo";
+    rev = "9f0b3315eed32409639a05aca55d7a0252681193";
+    sha256 = "sha256-Lr4+bXdVxYbCXKVzE+fjeLD559HuABK6lOLJ0sBBGNY=";
+  };
+  vendorSha256 = "sha256-T6PSs5NfXSXvzlq67rIDbzURyA+25df3nMMfufo0fow=";
```

Comment:
```suggestion
  version = "unstable-2022-08-25";

  src = fetchFromGitHub {
    owner = "digitallyserviced";
    repo = "tdfgo";
    rev = "9f0b3315eed32409639a05aca55d7a0252681193";
    sha256 = "sha256-Lr4+bXdVxYbCXKVzE+fjeLD559HuABK6lOLJ0sBBGNY=";
  };

  vendorSha256 = "sha256-T6PSs5NfXSXvzlq67rIDbzURyA+25df3nMMfufo0fow=";
```

Reply:
I applied this blindly.  I'm not sure what makes it `unstable`, or if there is additional ceremony for an unstable package.

Reply:
unstable-XXXX-XX-XX is the date of the commit since we are not setting the version to the commit hash or invent some new version number.